app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
+        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("outdated")
+        elif not vstr["wearing_a_mask"]:
+            raise NotWearingMaskError("not wearing mask")
+        else:
```

Comment:
check checklist
https://github.com/NEkropo1/cafe-test-task-pt/blob/master/checklist.md

It would be much better to read more descriptive names and error messages, just check how this code looks like:
```python
    def visit_cafe(self, visitor_dict: dict) -> str:
        if "vaccine" not in visitor_dict.keys() or visitor_dict["vaccine"] is None:
            raise NotVaccinatedError("Visitor not vaccinated.")
        elif visitor_dict["vaccine"]["expiration_date"] < datetime.date.today():
            raise OutdatedVaccineError("Visitor's vaccine is outdated.")
        elif not visitor_dict["wearing_a_mask"]:
            raise NotWearingMaskError("Visitor not wearing a mask.")
        
        return f"Welcome to {self.name}"
 ```